src/http-common.js

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios';
+
+const APP_ID = 'a1cpw2l4oEIk9qbS2aPvGfq';
+export const API_URL = `https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/${APP_ID}/books`;
+
+export default axios.create({
+  baseURL: API_URL,
+});
```

Comment:
![API](https://user-images.githubusercontent.com/93039359/209286378-d000c153-3c6d-46a3-b9e6-6a577a3602c7.PNG)
- [ ] It's great that you have implemented the api but it is not working properly 😨 as you can see in the image above ⬆️ Kindly fix this issue 👍 There might be a problem in your base url 🤔